aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -138,12 +139,14 @@ Tensor qnnpack_add(Tensor qa, Tensor qb, double scale, int64_t zero_point) {
   const auto a_scale = qa_contig.q_scale();
   const auto b_scale = qb_contig.q_scale();
 
-  Tensor qy = at::_empty_affine_quantized(
+  Tensor qy = at::new_qtensor_cpu(
```

Comment:
could you use at::native:: empty_affine_quantized?

Reply:
Whey do you think that would be better? Does it get around the dispatch overhead?

Reply:
It is better because it has the same API as at::_empty_affine_quantized.
yes, it will get around the dispatch overhead.

Reply:
Sure. Let me try.

Reply:
This is still slower, but not a whole lot. So I think it can make for a decent compromise.